src/pages/shooter/index.tsx

```diff
@@ -43,20 +48,24 @@ const Shooter = () => {
 	useEffect(() => {
 		onMessage((data) => {
 			if (data.message_type === MessageType.Hit && data.id === uuid) {
-				setScore((prevScore) => prevScore + 1);
-				console.log(score);
+				const newScore = score + 1;
+				setScoreState(newScore);
```

Comment:
```suggestion
				setScoreState((prev) => prev + 1);
```
こんな感じで値の更新をしないと正しくstateが更新されない場合があるので、変更お願いします